src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/incidents/IncidentService.kt

```diff
@@ -36,10 +35,26 @@ class IncidentService(
 
   fun findIdsByFilter(pageRequest: Pageable, incidentFilter: IncidentFilter): Page<IncidentIdResponse> {
     log.info("Incident Id filter request : $incidentFilter with page request $pageRequest")
-    return incidentRepository.findAll(IncidentSpecification(incidentFilter), pageRequest)
-      .map { IncidentIdResponse(it.id) }
+    return findAllIds(
+      fromDate = incidentFilter.fromDate?.atStartOfDay(),
```

Comment:
very minor but i guess that Date filter should indeed just be dates and now date times?